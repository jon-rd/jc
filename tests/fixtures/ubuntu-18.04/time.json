{"user_time":0.0,"system_time":0.0,"elapsed_time":"0:00.00","cpu_percent":100,"average_shared_text":"0","average_unshared_data_size":0,"maximum_resident_set_size":2064,"block_input_operations":0,"block_output_operations":0,"major_pagefaults":0,"minor_pagefaults":71,"swaps":0,"elapsed_time_hours":0,"elapsed_time_minutes":0,"elapsed_time_seconds":0,"elapsed_time_centiseconds":0,"elapsed_time_total_seconds":0.0}
