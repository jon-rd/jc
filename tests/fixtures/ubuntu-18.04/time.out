0.00user 0.00system 0:00.00elapsed 100%CPU (0avgtext+0avgdata 2064maxresident)k
0inputs+0outputs (0major+71minor)pagefaults 0swaps

